naming/src/main/java/com/alibaba/nacos/naming/consistency/ephemeral/distro/component/DistroHttpAgent.java

```diff
@@ -35,8 +36,17 @@
  */
 public class DistroHttpAgent implements DistroTransportAgent {
     
+    private final ServerMemberManager memberManager;
```

Comment:
And can we transfer a static method to check targetServer  in memberManager, the logic maybe used in other place. When we want to use the logic, didn't inject memberManager.

Reply:
Too Many static method and object will caused over-coupling, and hard to modified and refactor